Userland/Shell/AST.cpp

```diff
@@ -107,10 +107,16 @@ ErrorOr<void> AK::Formatter<Shell::AST::Command>::format(FormatBuilder& builder,
 namespace Shell::AST {
 
 template<typename... Args>
-static inline void print_indented(int indent, CheckedFormatString<Args...> format, Args&&... args)
+static inline ErrorOr<void> print_indented(int indent, CheckedFormatString<Args...> format, Args&&... args)
```

Comment:
I'm honestly not a fan of making debug paths TRY() things, but sure, whatever.

Reply:
Well all callers are TRY, but wouldnt actually throw, just call print_indented